### PR TITLE
Deff changes

### PR DESCRIPTION
**Pull request by Duny-**
I decided to come back to "completely ignore valid criticism"(©Rei[randomnumbers], 2015) once again. Changelog included.

ss 2015-09-16 at 02 05 48
Yes I am aware of that one pipe. Gonna have to redo the entire piping if deff sees any play one day anyway, it's horrible and really fucking inefficient for a map called efficiency.
ss 2015-09-16 at 02 06 34
ss 2015-09-16 at 02 08 35
Should fix or at least improve the 3 issues on gitlab. 
Also fixes https://github.com/d3athrow/vgstation13/issues/5097
fixes https://github.com/d3athrow/vgstation13/issues/3785
closes https://github.com/d3athrow/vgstation13/issues/3627 I don't plan on making the brig floodable with n2o, sounds like something you should ask an atmos tech to install for you

Also this is working as intended, closes https://github.com/d3athrow/vgstation13/issues/4494, if you want paramedics doing their job play paramedic like I used to
This is obsolete, closes https://github.com/d3athrow/vgstation13/issues/2511
N3X15 casualized the supermatter setup a bit (I like it tho) so this should be closed too, closes https://github.com/d3athrow/vgstation13/issues/2514

Tried not to forget to add lights this time
Boop

### CODE REVIEW

**Comment by N3X15** [comment]
GitHub checks for shit like "closes #1", so put those in there so they're autoclosed.

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -829,6 +829,10 @@ proc/process_adminbus_teleport_locs()
 	name = "Abandoned Diner"
 	icon_state = "dinerderelict"
 
+/area/derelictparts/incinerator
```
Comment (N3X15) [inline code/game/area/Space Station 13 areas.dm]: Move to maps/defficiency/areas.dm so all the map shit is in one bundle.  Use ancientstation.dm as an example on how to properly include the *.dm's.

**Comment by Duny-** [comment]
Like dis?

**Comment by N3X15** [comment]
Yep, that's good.  I'm trying to remove the cruft that's accumulated in SS13 areas.dm over the years.

**Pull request merged**